web/ce/hooks/use-page-flag.ts

```
@@ -4,11 +4,13 @@ export type TPageFlagHookArgs = {
 
 export type TPageFlagHookReturnType = {
   isMovePageEnabled: boolean;
+  isPageSharingEnabled: boolean;
 };
 
 export const usePageFlag = (args: TPageFlagHookArgs): TPageFlagHookReturnType => {
   const {} = args;
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)